src/handlers/clean/business-logic.ts

```diff
@@ -0,0 +1,39 @@
+import { BusinessLogic } from "../../handler-context";
+import { fetchVendorId } from "../../shared/utils/config-utils/fetch-vendor-id";
+import { CleanedEventBody, Env, IncomingEventBody } from "./types";
+
+export const businessLogic: BusinessLogic<
+  IncomingEventBody,
+  Env,
+  never,
+  CleanedEventBody
+> = async (
+  {
+    component_id,
+    event_id,
+    event_name,
+    timestamp,
+    timestamp_formatted,
+    user,
+    vendor_id,
+  },
+  { logger }
+) => {
+  const vendorId = vendor_id ?? (await fetchVendorId(event_name));
```

Comment:
Since `fetchVendorId` fetches config could we rewrite it as a sync function that takes the config as an argument and pass in a config from the context here? It looks like this is the only place that `fetchVendorId` is used so shouldn't cause duplication